.github/workflows/test.yml

```diff
@@ -23,6 +12,5 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 20
-      - run: npm install --frozen-lockfile
-      - run: npm run build
-      - run: npm run lint
+      - run: yarn install --frozen-lockfile
+      - run: yarn run validate-change
```

Comment:
```suggestion
      - run: yarn run validate-change
      - run: yarn run build
```

Reply:
Can we also add `build` separately? Or as part of validate change? I think there are scenarios in which TS build can pass but next build may fail, just want to cover those!